tools/test-static-code

```diff
@@ -122,7 +122,7 @@ else
     # all normal files which are either:
     #   - filename is *.py
     #   - executable matching ^#!/usr/bin/python3
-    PYFILES="$(find . -type f '(' -name '*.py' -o -executable -execdir grep -q '^#!/usr/bin/python3' '{}' ';' ')'  -print)"
+    PYFILES="$(git grep -lI '^#!.*python3') $(git ls-files "*.py")"
```

Comment:
This ought to be guarded by `if [ -e .git ]` like in "valid JSON files"`, as we run test-static-code during `make check`. It is entirely plausible to build a released cockpit in a minimal build environment without `git` being installed.

Reply:
Good idea.  Thanks!